File: python/average array.py
#Write a python program to calculate the average value of array elements. 


num=int(input('how many numbers you want: '))
list=[]
for i in range(num):
    enter=int(input('enter the number: '))
    list.append(enter)

avg_list=sum(list)/len(list)

print('the array is: ',list)
print('the average of elements are: ',float(avg_list))

File: Java_Programs/Array/averageArray.java
import java.util.*;
import java.util.Scanner;
public class average_Array {
	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		System.out.print("enter any number: ");
		int n=sc.nextInt();
		int a[]=new int[n];
		double sum=0;
		for(int i=0;i<n;i++) {
			a[i]=sc.nextInt();
			sum+=a[i];
		}
		System.out.println("input array is: "+Arrays.toString(a));
		double avg=(sum)/n;
		System.out.println("average of elements in array is: "+avg);
	}
}
